designsafe/static/scripts/projects/components/edit-project/edit-project.component.js

```diff
@@ -96,7 +96,7 @@ class EditProjectCtrl {
             this.form.description = this.project.value.description || '';
             this.form.experimentalFacility = this.project.value.experimentalFacility || '';
             this.form.keywords = this.project.value.keywords || '';
-            this.form.dataType = this.project.value.dataType || '';
+            this.form.dataType = this.project.value.dataType || null;
```

Comment:
any reason this needed to be `null`?

Reply:
This was to prevent a blank default value from being displayed in a dropdown of data types. If this is defined as an empty string, our first option will be a blank dropdown bar. If If we set this to `null` We'll be able to use a default value `<option value="" disabled selected>--Data Type--</option>` to display "--Data Type--" within the ng-options of data types when the field is incomplete.